official/utils/logging/examples_per_second_hook.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
```

Comment:
Have you run linted this code? I think there should be two newlines here, one below-- make sure to lint.